infra/main.bicep

```diff
@@ -174,10 +180,14 @@ module backend 'core/host/appservice.bicep' = {
       OPENAI_ORGANIZATION: openAiApiOrganization
       // Optional login and document level access control system
       AZURE_USE_AUTHENTICATION: useAuthentication
+      AZURE_ENFORCE_ACCESS_CONTROL: enforceAccessControl
       AZURE_SERVER_APP_ID: serverAppId
       AZURE_SERVER_APP_SECRET: serverAppSecret
       AZURE_CLIENT_APP_ID: clientAppId
-      AZURE_TENANT_ID: tenant().tenantId
+      AZURE_CLIENT_APP_SECRET: clientAppSecret
```

Comment:
This should become a keyvault reference instead.

Reply:
will update when we have keyvault support